src/radical/pilot/utils/misc.py

```diff
@@ -157,9 +157,12 @@ def get_resource_fs_url(resource: str,
     For the given resource label, return the contact URL of the resource's file
     system.
 
+    This URL is equivalent to the base ``endpoint:///`` URL available for use
```

Comment:
I have not confirmed yet if this is correct. But whatever the correct mapping is, documenting it would resolve one of the use cases documented for #2668

Reply:
This is correct.